src/dialog/index.ts

```diff
@@ -77,12 +90,8 @@ export class Dialog extends KucBase {
             ${this._getCloseButtonSvgTemplate()}
           </button>
         </div>
-        <div class="kuc-dialog__dialog__content">
-          ${unsafeHTML(cleanContent)}
-        </div>
-        <div class="kuc-dialog__dialog__footer">
-          ${unsafeHTML(cleanFooter)}
-        </div>
+        <div class="kuc-dialog__dialog__content"></div>
+        <div class="kuc-dialog__dialog__footer"></div>
```

Comment:
Display the content and footer here
```suggestion
        <div class="kuc-dialog__dialog__content">${this.content}</div>
        <div class="kuc-dialog__dialog__footer">${this.footer}</div>
```